src/services/jsDoc.ts

```diff
@@ -1,48 +1,92 @@
 /* @internal */
 namespace ts.JsDoc {
     const jsDocTagNames = [
+        "abstract",
+        "access",
+        "alias",
+        "arg",
+        "argument",
+        "async",
         "augments",
         "author",
-        "argument",
         "borrows",
         "callback",
         "class",
+        "classDesc",
+        "const",
         "constant",
         "constructor",
         "constructs",
+        "copyright",
         "default",
         "deprecated",
         "description",
+        "emits",
+        "enum",
         "event",
         "example",
+        "exports",
         "extends",
+        "external",
         "field",
+        "file",
         "fileOverview",
+        "fires",
+        "func",
         "function",
+        "generator",
+        "global",
+        "hideConstructor",
+        "host",
         "ignore",
+        "implements",
         "inheritDoc",
         "inner",
+        "instance",
+        "interface",
+        "kind",
         "lends",
+        "license",
         "link",
+        "linkCode",
+        "linkPlain",
+        "listens",
+        "member",
         "memberOf",
         "method",
+        "mixes",
+        "module",
         "name",
         "namespace",
+        "override",
+        "package",
         "param",
         "private",
         "prop",
         "property",
+        "protected",
         "public",
+        "readonly",
         "requires",
+        "return",

```

Comment:
Part of my concern is that the tag list lists aliases that will be frustrating for users who are used to `returns`.